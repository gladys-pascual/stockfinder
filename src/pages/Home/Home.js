import CurrentNews from "../../components/CurrentNews/CurrentNews";
import Search from "../../components/Search/Search";
import "./Home.scss";

const Home = () => {
  return (
    <div className="home">
      <Search />
      <CurrentNews />
    </div>
  );
};

export default Home;
