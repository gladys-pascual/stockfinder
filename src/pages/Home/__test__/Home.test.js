import { render } from "@testing-library/react";
import Home from "../Home";

test("render Home component as expected", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
