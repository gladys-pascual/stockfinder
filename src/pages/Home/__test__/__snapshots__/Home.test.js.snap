// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Home component as expected 1`] = `
<div>
  <div
    class="home"
  >
    <form
      class="search-form"
    >
      <div
        class="form-wrapper"
      >
        <input
          class="search-input"
          name="query"
          placeholder="Search for symbol or company name..."
          type="text"
        />
        <div
          class="logo-search"
        >
          <img
            alt="Logo of StockFinder"
            src="StockFinderLogo.svg"
          />
        </div>
      </div>
    </form>
    <div
      class="query-results-list"
    >
      <ul
        class="results"
      />
    </div>
  </div>
</div>
`;
